roles/splunk_standalone/molecule/default/playbook.yml

```diff
@@ -27,7 +27,7 @@
       license_uri: null
       wildcard_license: false
       build_remote_src: true
-      build_location: ${SPLUNK_INSTALLER}
+      build_location: https://download.splunk.com/products/splunk/releases/7.3.0/linux/splunk-7.3.0-657388c7a488-Linux-x86_64.tgz
```

Comment:
What's the reasoning here? Won't we have to manually update this now?

Reply:
This is just used for the molecule tests. The previous setting of `${SPLUNK_INSTALLER}` actually doesn't work anymore - I'm guessing that's an artifact of our old image pattern? I couldn't find any references to a default env var `SPLUNK_INSTALLER` in docker-splunk.

Reply:
And no, we shouldn't need to update this unless the way we build the image changes again - this is just used for tests so it should dynamically fetch the 7.3.0 build and setup the container using that.